control-plane/pkg/security/secret.go

```diff
@@ -220,3 +220,11 @@ func skipClientAuthCheck(data map[string][]byte) (bool, error) {
 	}
 	return enabled, nil
 }
+
+func firstNonEmptyValueOf(data map[string][]byte, keys ...string) ([]byte, bool) {
+	userKeyOrCert, ok := data[keys[0]]
+	if !ok {
+		userKeyOrCert, ok = data[keys[1]]
+	}
+	return userKeyOrCert, ok
+}
```

Comment:
We shouldn't assume only 2 keys, so we need a for loop here

Reply:
Something like this:

```golang
func firstNonEmptyValueOf(data map[string][]byte, keys ...string) ([]byte, bool) {
	for _, key := range keys {
	    if v, ok := data[key]; ok {
	        return v, ok
	     }
	}
	return "", false
}
```

Reply:
Done! 👍🏽 , please Review it